xmtp_debug/Cargo.toml

```diff
@@ -22,6 +22,7 @@ fs_extra = "1.3"
 hex.workspace = true
 indicatif = "0.17"
 k256 = "0.13"
+lazy_static = "1.5.0"
```

Comment:
rust standard library has a [LazyCell](https://doc.rust-lang.org/std/cell/struct.LazyCell.html) and [LazyLock](https://doc.rust-lang.org/std/sync/struct.LazyLock.html) that replaces uses of lazy static